plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClient.java

```diff
@@ -879,4 +889,84 @@ private static ResultSet getResultSet(PhoenixInputSplit split, QueryPlan queryPl
             throw new TrinoException(PhoenixErrorCode.PHOENIX_INTERNAL_ERROR, "Error while copying scan", e);
         }
     }
+
+    public JdbcTableHandle buildPlainTable(JdbcTableHandle handle)
+    {
+        checkArgument(handle.isNamedRelation(), "Only allow build plain table from named relation table");
+
+        SchemaTableName schemaTableName = handle.getRequiredNamedRelation().getSchemaTableName();
+        RemoteTableName remoteTableName = handle.getRequiredNamedRelation().getRemoteTableName();
+        return new JdbcTableHandle(schemaTableName, remoteTableName, Optional.empty());
+    }
+
+    public JdbcTableHandle updatedScanColumnTable(ConnectorSession session, JdbcTableHandle tableHandle, Optional<List<JdbcColumnHandle>> originalColumns, JdbcColumnHandle mergeRowIdColumnHandle)
```

Comment:
Nit: Does `originalColumns` need to be `Optional`?   Since it is `Optional<List<JdbcColumnHandle>` wouldn't passing the list be preferable?

Reply:
The `originalColumns` can come from the `JdbcTableHandle`, the value can not exists in general. May be we can leave this here? what do you think